src/hotspot/share/opto/castnode.cpp

```diff
@@ -26,11 +26,13 @@
 #include "opto/addnode.hpp"
 #include "opto/callnode.hpp"
 #include "opto/castnode.hpp"
+#include "opto/cfgnode.hpp"
 #include "opto/connode.hpp"
 #include "opto/matcher.hpp"
 #include "opto/phaseX.hpp"
 #include "opto/subnode.hpp"
 #include "opto/type.hpp"
+#include "opto/loopnode.hpp"
```

Comment:
The new unsorted include now causes `sources/TestIncludesAreSorted.java` to fail.